packages/dataviews/src/types.ts

```diff
@@ -247,6 +247,12 @@ export interface NormalizedFilter {
 	isPrimary: boolean;
 }
 
+export enum DensityOptions {
+	medium = 0,
+	compact = 1,
+	comfortable = 2,
```

Comment:
In practice it doesn't matter, as the consumers would never have to know the actual values of the enum. I'll update though to have `strings` (like `compact = 'compact'), because in most cases density is something that would be handled in css and consumers should just add a wrapper ccs class name (like `is-density-${densityOption}`.